go/pkg/codes/constants_gen.go

```diff
@@ -61,11 +61,15 @@ const (
 
 	// Permission
 
+	// Duplicate indicates the requested identity already exists.
+	UnkeyDataErrorsPermissionDuplicate URN = "err:::"
```

Comment:
These two did not generate

Reply:
fixed, I forgot to add the field to the UnkeyDataErrors struct